### ISSUE TITLE
Attempting to Extract Individual Voice Lines

### ISSUE THREAD

**Issue by Pentakellium** [r-ex/rsx#23]
Hey all, brand new here, found the Twitter page from a pretty old reddit post directing people to Legion, and was wondering what the process of getting Voice Lines is supposed to be like, if it is even still possible. Not too keen on downloading hour long voice line compilations just to get the one or two i want.

**Comment by biast12** [comment]
It's not possible to do with RSX at the moment, but it hopefully will be added at some point, rn is the only way to do it but using this specific legion+ version https://biast12.site/Apex_PC_Archive/_Tools_for_Getting_the_files/LegionLatest/LegionPlusAudio.exe

**Comment by IJARika** [comment]
please don't link executable files, thanks. the most recent version of legion can be built from or downloaded from [](https://github.com/r-ex/LegionPlus), which can be used to download the audio.

**Comment by biast12** [comment]
No, the latest version of LegionPlus can't export voicelines that work (they're all broken)...., that's why i linked to the only version that does (from Rexx even), the reason why is because they added some rad encoding which isn't included in the source code (for obvious reasons) = anyone can't just built it, especially not without knowing how to

**Comment by IJARika** [comment]
do not link unverified software, thank you.

**Comment by biast12** [comment]
I was from rexx.......
